OrbitLinuxTracing/include/OrbitLinuxTracing/Tracer.h

```diff
@@ -73,11 +77,7 @@ class Tracer {
   std::vector<Function> instrumented_functions_;
 
   TracerListener* listener_ = nullptr;
-
-  bool trace_context_switches_ = true;
-  bool trace_callstacks_ = true;
-  bool trace_instrumented_functions_ = true;
-  bool trace_gpu_driver_ = true;
+  TracingOptions tracing_options_;
```

Comment:
But now the defaults are moved from there to `TracingOptions.h`.

Reply:
Still, I think this is an improvement. As there are all grouped in TracingOptions.h

Reply:
Mmh, maybe you can initialize the fields explicitly?
`TracingOptions tracing_options_{true, kDwarf, true, true};`
Although that does look a bit unreadable without comments with the names of the fields (without an IDE showing them).

Reply:
How about named initialization:
```
  TracingOptions tracing_options_{.trace_context_switches = true,
                                  .sampling_method = kDwarf,
                                  .trace_instrumented_functions = true,
                                  .trace_gpu_driver = true};
```